installer/alpine/init-aarch64

```diff
@@ -312,6 +315,11 @@ rescue)
 esac
 eend
 
+discover_address="tinkerbell.$KOPT_facility.packet.net"
+if [ "$packet_action" == "discover" ] && [ -n "$KOPT_discover_address" ]; then
+  discover_address=$KOPT_discover_address
+fi
```

Comment:
Not sure this is necessary since we're grepping it out of /proc/cmdline in the discover.sh file

Reply:
Ah, good catch. I had refactored this and forgot it was still in init-aarch64.